### ISSUE TITLE
False positive SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName (regression)

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1258]
```php
<?php declare(strict_types = 1);

namespace Bug;

use NumberFormatter;
use OutOfBoundsException;

static function (NumberFormatter $formatter, string $attribute, mixed $value): void {
    if (!$formatter->setAttribute($attribute, $value)) {
        throw new OutOfBoundsException(
            "\\NumberFormatter::setAttribute() failed: '{$attribute}' is unknown/invalid.",
        );
    }
};
```

```xml
<?xml version="1.0"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <config name="installed_paths" value="../../slevomat/coding-standard"/>
    <rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly"/>
</ruleset>
```

```
phpcs -- packages/Bug.php
> @php ./vendor/bin/phpcs 'packages/Bug.php'

FILE: /project/packages/Bug.php
-----------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------------------
 11 | ERROR | [x] Class \NumberFormatter should not be referenced via a fully qualified name, but via a use statement.
-----------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------

Time: 645ms; Memory: 14MB
```

```
name     : slevomat/coding-standard
descrip. : Slevomat Coding Standard for PHP_CodeSniffer complements Consistence Coding Standard by providing sniffs with additional checks.
keywords :
versions : * dev-master, * 7.x-dev
type     : phpcodesniffer-standard
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
homepage :
source   : [git] https://github.com/slevomat/coding-standard.git db243812192ab2308098dfeec5ea8a95b93aba18
dist     : [zip] https://api.github.com/repos/slevomat/coding-standard/zipball/db243812192ab2308098dfeec5ea8a95b93aba18 db243812192ab2308098dfeec5ea8a95b93aba18
path     : /project/vendor/slevomat/coding-standard
names    : slevomat/coding-standard
```

**Comment by kukulich** [comment]
As a hotfix, you can problably use `'` instead of `"`
